addon/components/frost-table.js

```diff
@@ -47,5 +59,20 @@ export default Component.extend({
   // == Actions ===============================================================
 
   actions: {
+    /**
+     * Wrap our arguments in a single object, so that cell renderers can trigger arbitrary events.
+     * Your handler is then responsible for doing stuff based on actions like 'click' or 'input'.
+     *
+     * @param {Number} row - data rows are zero based, and header has row -1
+     * @param {Number} col - column index
+     * @param {String} action - this comes after row/col as Ember lets us include those in action closures easily.
+     * @param {Object[]} args - any additional data
+     */
+    handleCallback (row, col, action, args) {
+      const handler = this.get('onCallback')
+      if (handler) {
```

Comment:
Once again, I don' think we want to. If we do, that's saying we don't trust prop-types and we should type-check everything we receive.